packages/react-core/src/filters/tileFeaturesSpatialIndex.js

```diff
@@ -0,0 +1,94 @@
+import { indexes } from '@mapbox/tile-cover';
+// h3-js has a known problem that does not allow us to use it in a web-worker. To solve
+// it we're overwriting the node_module package after install it (check postinstall script in package.json)
```

Comment:
I would mention the `patches/h3-js+3.7.2.patch` file here. Until I found that I had no idea how this worked